<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchCaptureDialog</class>
 <widget class="QDialog" name="BatchCaptureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>117</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>批量快照</string>
  </property>
  <property name="windowIcon">
   <iconset resource="cc-client.qrc">
    <normaloff>:/Icons/capture.png</normaloff>:/Icons/capture.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>保存到：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="saveDirLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="selectDirPushButton">
       <property name="text">
        <string>选择位置</string>
       </property>
       <property name="icon">
        <iconset resource="cc-client.qrc">
         <normaloff>:/Icons/books_3.ico</normaloff>:/Icons/books_3.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startPushButton">
       <property name="text">
        <string>开始截图</string>
       </property>
       <property name="icon">
        <iconset resource="cc-client.qrc">
         <normaloff>:/Icons/52design.com_jingying_059.png</normaloff>:/Icons/52design.com_jingying_059.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="saveToOneDirCheckBox">
     <property name="text">
      <string>保存到同一个文件夹（否则每个截图将保存到工作站名称为名的文件夹下）</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string>点击开始截图开始截取...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="cc-client.qrc"/>
 </resources>
 <connections/>
</ui>
